src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure {
+    public static final int MAXIMUM_SIZE = 50;
+    private int radius;
+    private Random random = new Random();
```

Comment:
The `Random` instance should not be a part of the `Circle` class. You should pass the value for the radius from outside, ideally from the `FigureSupplier` class.